packages/core/test/app/ovm/integer-quantifiers.spec.ts

```diff
@@ -0,0 +1,28 @@
+/* Internal Imports */
+import {
+  IntegerRangeQuantifier,
+  NonnegativeIntegerLessThanQuantifier,
+} from '../../../src/app'
+
+describe('IntegerQuantifiers', () => {
+  describe('IntegerRangeQuantifier', () => {
```

Comment:
Would also test where `start === end`, when both are negative, when just one is negative, when `end < start`, etc.